server/db/index.js

```diff
@@ -10,9 +10,13 @@ var Tag = require('./models/tag');
 // if we had more models, we could associate them in this file
 // e.g. User.hasMany(Reports)
 User.belongsToMany(User, {as: 'friend', through: 'user_friends', foreignKey: 'friendId'});
+User.belongsToMany(User, {as: 'user', through: 'user_friends', foreignKey: 'userId'});
```

Comment:
Users have users? I get the `friend` association, but why the `user` association? @noeladd